try:
 resultado = 10 / 0
except:
 print("ocorreu um erro durante a operaÃ§ao")